src/Silverback.Integration/Messaging/Configuration/ConsumerEndpointBuilderDecryptUsingExtensions.cs

```diff
@@ -49,5 +50,48 @@ public static class ConsumerEndpointBuilderDecryptUsingExtensions
 
             return (TBuilder)endpointBuilder;
         }
+
+        /// <summary>
+        ///     Specifies that the AES algorithm has to be used to decrypt the messages.
+        /// </summary>
+        /// <typeparam name="TBuilder">
+        ///     The actual builder type.
+        /// </typeparam>
+        /// <param name="endpointBuilder">
+        ///     The endpoint builder.
+        /// </param>
+        /// <param name="key">
+        ///     The secret key for the symmetric algorithm. (In case of Key Rotation this is the default key).
```

Comment:
_"in case of key rotation"_ doesn't make sense. I mean, this overload is only used if you are rotating keys, because the `decryptionKeyCallback` is not optional.

In any case, I would really just have the callback to avoid confusion. The default can be implemented in the function itself.

Reply:
You're right, I fixed it!